verify/Privacy

```diff
@@ -1246,16 +1246,15 @@ Privacy-43: 「Do Not Track」ヘッダの送出の可否
 
     - 「プライバシーとセキュリティ」→「ウェブサイトに"Do Not Track"要求を送る」チェックボックスがONである。
 
-    :3: プライベートウィンドウのみ送出
+    :3: [廃止] プライベートウィンドウのみ送出
 
-    OPTIONS
-
-    - 「プライバシーとセキュリティ」→「ウェブサイトに"Do Not Track"要求を送る」チェックボックスがOFFである。
+    [廃止]
 
     :4: 送出しない
 
     OPTIONS
 
+    - 「プライバシーとセキュリティ」→「強化型トラッキング防止機能」→「カスタム」→「トラッキングコンテンツ」チェックボックスがOFFである。
```

Comment:
Checking this looks good but I'm concerned that it might be duplicated with that of Privacy-33-5.
I suppose it would be general for the verification for ESR update to skip the check that has already been verified for other configuration.

Reply:
I considered again to end up with changing my mind.
This check should be existent as it could work good to detect the configuration for Privacy-33 being other than 5.

Reply:
BTW, `cat-verify` should support dependencies of configuration items, for this kind cases.